compiler/rustc_typeck/src/check/wfcheck.rs

```diff
@@ -1086,15 +1086,8 @@ fn check_method_receiver<'fcx, 'tcx>(
 
     debug!("check_method_receiver: sig={:?}", sig);
 
-    let self_ty = fcx.normalize_associated_types_in(span, self_ty);
```

Comment:
I think this line got removed unintentionally.

Reply:
Oops, yes, restored.